LocalMultiplayerAgent/SettingsJsonGenerator/scripts.js

```diff
@@ -0,0 +1,127 @@
+let BuildGuId = crypto.randomUUID();
+let SessionGuId = crypto.randomUUID()
+
+/// DO NOT CHECK IN:
+/// https://playfab.visualstudio.com/PlayFab/_wiki/wikis/Thunderhead/1299/Run-LocalMultiplayerAgent
+/// Deep details about how each field works
+/// DO NOT CHECK IN
+
+function popAlert(){
```

Comment:
should we remove this?

Reply:
... yes. That's why we have reviewers! Thanks!